_health-care/_js/_components/date_input.jsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import _ from 'lodash';
+
+class DateInput extends React.Component {
+  constructor() {
+    super();
+    this.state = {month: '', date: '', year: ''};
```

Comment:
Nevermind! Read more of the docs and sounds like this is just the es6 way of doing things :)